doc/iterations/2025.8/add_support_for_exporting_query_result.adoc

```diff
@@ -0,0 +1,55 @@
+= Add support for exporting query result
+
+== Problem
+
+Users can't export the result of a query from the query view.
+
+== Key Result
+
+- A button should be available on the query view to export the result
+- This button should present the export options the user can select
+- An implementation to export results as CSV is provided, the CSV file should contain the following information
+* ID of the element
+* Type of the element
+* Label of the element (if any)
+- When pressed, a file containing the result should be automatically downloaded by the user's browser
+- The button should not be enabled if the _evaluation result_ view is empty or if it does not contain objects
```

Comment:
Is this restriction fine with you (only support objects for now)?
This way we can provide a single API endpoint to download objects, and not bother with other kind of results.

Reply:
fixed